src/extlibs/hrvo/hrvo_agent.cpp

```diff
@@ -425,42 +422,51 @@ void HRVOAgent::computeNewVelocity()
         return -1;
     };
 
-    // Choosing a candidate point has these goals:
+    // Choosing a candidate velocity has these goals:
     // - Pick a velocity as close as possible to the preferred velocity as long as it
-    // doesn't lie in any velocity obstacle.
-    // - Failing the first condition, then choose the best preferred velocity that
-    // minimizes collisions with the closest velocity obstacle
-    // - Candidate multimap is organized by distance from preferred velocity so all we
-    // have to do is pick the first velocity that is valid
-    for (auto [dist_sq, candidate] : candidates_)
+    // doesn't lie in any velocity 	 	obstacle
+    // - Failing the first condition, then choose the best candidate velocity as new
+    // velocity that minimizes collisions 		with the closest velocity obstacles
```

Comment:
nit, there seems to be some random spaces in the comment. Not sure if its accident or done by formatting script.

Reply:
no clue why those exist, but should be fixed now